packages/dds/merge-tree/src/snapshotLoader.ts

```diff
@@ -102,8 +102,16 @@ export class SnapshotLoader {
             if (spec.removedSeq !== undefined) {
                 seg.removedSeq = spec.removedSeq;
             }
+            // this format had a bug where it didn't store all the overlap clients
+            // this is for back compat, so we change the singular id to an array
+            // this will only cause problems if there is an overlapping delete
+            //spanning the snapshot, which should be rare
             if (spec.removedClient !== undefined) {
```

Comment:
this was another bug where we missed the overlap clients in the snapshot

Reply:
My mistake.  Thanks for fixing!